src/FirebirdSql.EntityFrameworkCore.Firebird.FunctionalTests/Query/Associations/OwnedTableSplitting/OwnedTableSplittingProjectionFbTest.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ *    The contents of this file are subject to the Initial
+ *    Developer's Public License Version 1.0 (the "License");
+ *    you may not use this file except in compliance with the
+ *    License. You may obtain a copy of the License at
+ *    https://github.com/FirebirdSQL/NETProvider/raw/master/license.txt.
+ *
+ *    Software distributed under the License is distributed on
+ *    an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either
+ *    express or implied. See the License for the specific
+ *    language governing rights and limitations under the License.
+ *
+ *    All Rights Reserved.
+ */
+
+//$Authors = Niek Schoemaker (@niekschoemaker)
+
+using System.Threading.Tasks;
+using FirebirdSql.EntityFrameworkCore.Firebird.FunctionalTests.Helpers;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Query.Associations.OwnedTableSplitting;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FirebirdSql.EntityFrameworkCore.Firebird.FunctionalTests.Query.Associations.OwnedTableSplitting;
+
+public class OwnedTableSplittingProjectionFbTest : OwnedTableSplittingProjectionRelationalTestBase<OwnedTableSplittingFbFixture>
+{
+	public OwnedTableSplittingProjectionFbTest(OwnedTableSplittingFbFixture fixture, ITestOutputHelper testOutputHelper) : base(fixture, testOutputHelper)
+	{
+		Fixture.TestSqlLoggerFactory.Clear();
+		Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
+	}
+
+	// Uses OUTER JOIN
```

Comment:
I think with #1238 this should be fine, no?

Reply:
Yes, that's correct, I was awaiting that pull request to be merged, that's why I added the comment in the first place to easily find those tests